Automattic/vip-go-mu-plugins
# Removing Enterprise Search UI

## Description

After the changes in https://github.com/Automattic/vip-go-mu-plugins/pull/2366, the Enterprise Search UI (Widget) is essentially dead code that's not called anywhere. 

This PR deletes that code. If it is needed in the future, we simply have to revert this PR.

## Changelog Description

### Removing Enterprise Search UI 

We removed the Enterprise Search UI Widget code.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Go to `wp-admin` > `Widgets`
3. You shouldn't see any breaking changes.
